asab/web/auth/utils.py

```
@@ -11,46 +12,71 @@
 	"""
 	authorization_header = request.headers.get(aiohttp.hdrs.AUTHORIZATION)
 	if authorization_header is None:
 		L.debug("No Authorization header.")
 		raise NotAuthenticatedError()
 
 	try:
-		auth_type, token_value = authorization_header.split(" ", 1)
+		auth_scheme, token_value = authorization_header.split(" ", 1)
 	except ValueError:
 		L.warning("Cannot parse Authorization header.")
 		raise NotAuthenticatedError()
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)